### ISSUE TITLE
Audit web components versions and decide if we opt-out of upgrade for existing v1s (by 2/9?)

### ISSUE THREAD

**Issue by buddhima-coforma** [department-of-veterans-affairs/va.gov-team#74170]
## Description
Announcement: https://dsva.slack.com/archives/C03R5SBELQM/p1705503589174489

_On February 16 all design system web components will default to their v3 variant. v3 will become opt out only. 
In one month, the design system team will be switching all web components to render their v3 variant automatically. If teams have a need to fall back to the original v1 variant, they will be able to do so by setting the uswds prop to false. However, v1 variants will eventually be fully retired and removed so opting out is discouraged._

## Tasks

1. Identify how many of our components are not set to V3, so we have time to make changes if needed (ideally by 2/9?)
2. Decide if it makes sense to set to V1 first and V3 later, or to go directly to V3 - what is the path of least resistance to avoid bad UX?
3. Make changes where needed (before 2/16)
4. Test and verify that form is working
5. Create tickets for any future work that needs to be done related to this

## Acceptance Criteria
- Form is usable after 2/16 and Veterans don't face any undue UX which hampers their form-filling and submission experience

**Comment by mjknight50** [comment]
From an engineering stand-point, I think we would only know if a component IS v3 if it has
USWDS: true

**Comment by vbellissimo** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/department-of-veterans-affairs/va.gov-team/74170?planning-poker) with Zenhub @ToddWebDev @TaiWilkin @danlim715 @scottsdevelopment @wayne-weibel

**Comment by aplatt-coforma** [comment]
Summary from Todd

> Good news! The web components audit for the Pension Benefits form reveals that we are not using any v1 web components and no work is required before web components will default to their v3 variant on February 16.
However, while we don't use any v1 web components in our UI, the audit revealed that we use many old UI elements in our form that aren't web components. For example, many of our pages in the marriages and dependents chapter are using widgets and fields that use the older React Json Schema Form (RJSF) elements that are deprecated.
We use 28 widgets throughout our form. Most of these widgets can be updated to use form web component patterns that have been updated to use v3 web components
We have 18 fields that can be updated to use web components
By updating to web components our form will be better synced to the design, styling, and functionality of the US System. I've summarized the details of my findings in this [google doc](https://docs.google.com/document/d/1alULw9483ToBSweyhu8VVIqf8-Yx57EDuVhjXfDF2cg/edit#heading=h.tcapkocmxmn4)
